web/fluidity.money/src/screens/MailingList/MailingList.tsx

```diff
@@ -5,8 +5,11 @@
 import { GeneralButton, Heading, Text } from '@fluidity-money/surfing';
 import { motion, useAnimation } from 'framer-motion';
 import useViewport from 'hooks/useViewport';
+import { useState } from 'react';
 import styles from './MailingList.module.scss';
 
+type FormState = `success` | `loading` | `failed` | `idle`;
```

Comment:
could just be normal quotes (`""`)

Reply:
yeah ik, just wanted to use those to identify that they are for state checks